dist/bin/common

```diff
@@ -133,14 +133,18 @@ fi
 # *-------------------------------------------------*/
 
 find_lib () {
-  local lib=$(find $PROG_HOME/lib/ -name "$1")
-  if [ -n "$CYGPATHCMD" ]; then
-    "$CYGPATHCMD" -am $lib
-  elif [[ $mingw ||  $msys ]]; then
-    echo $lib | sed 's|/|\\\\|g'
-  else
-    echo $lib
-  fi
+  for lib in "$PROG_HOME"/lib/$1 ; do
+    if [[ -e "$lib" ]]; then
```

Comment:
```suggestion
    if [[ -f "$lib" ]]; then
```

As pointed out by @michelou (thanks!), we can make this more robust by using `-f` here.